bundles/EcommerceFrameworkBundle/IndexService/Worker/Helper/MySql.php

```diff
@@ -141,7 +141,7 @@ public function createOrUpdateIndexStructures()
                 Logger::info($e);
             }
 
-            $this->dbexec('ALTER TABLE `' . $this->tenantConfig->getTablename() . '` ENGINE = MyISAM;');
+            $this->dbexec('ALTER TABLE `' . $this->tenantConfig->getTablename() . '` ENGINE = ' . $this->tenantConfig->getTableEngine() . ';');
```

Comment:
you also have to check here if the tenantConfig implements the IMysqlEngineConfig